packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -15,21 +40,46 @@ export const AdvancedSettings = ({
 }: {
   destinationAddress?: string
   onChange: (value?: string) => void
-  error: AdvancedSettingsErrors | null
+  error: DestinationAddressErrors | null
 }) => {
   const {
-    app: { selectedToken }
+    app: { selectedToken, isDepositMode }
   } = useAppState()
+  const { l1, l2 } = useNetworksAndSigners()
```

Comment:
We want to stop using `useNetworksAndSigners` but in this case we don't have a different way of getting L2 provider?